node-playwright/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM mcr.microsoft.com/playwright:focal
+ARG PLAYWRIGHT_VERSION=
+FROM mcr.microsoft.com/playwright:${PLAYWRIGHT_VERSION}jammy
```

Comment:
Isn't this missing some `:` or `-` after the pw version?

Reply:
Nop, thats fine, it's added in the workflow

Reply:
Oh right, so it can work with no version specified, now I get it